File: sdk/client/java/openapi-test/src/test/java/opensdk/sdk/apis/klay/account/KlayEncodeAccountKeyApiTest.java
package opensdk.sdk.apis.klay.account;

import opensdk.sdk.apis.constant.UrlConstants;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;
import org.klaytn.OpenSDK;

import java.io.IOException;

@DisplayName("Klay RPC Test")
public class KlayEncodeAccountKeyApiTest {

    private final OpenSDK sdk = new OpenSDK(UrlConstants.TEST_URL);

    @Test
    @DisplayName("RPC klay_encodeAccountKey")
    void whenRequestValid_ThenCall200ResponseReturns() throws IOException {
        //todo
    }
}

File: web3rpc/sdk/client/javascript/openapi-test/test/klay/account/KlayEncodeAccountKeyApi.test.js
const OpenSdk = require("@klaytn/web3rpc");
const { expect } = require("@jest/globals");
const { BAOBAB_RPC } = require("../../constant");

const sdk = new OpenSdk.KlayApi(new OpenSdk.ApiClient(BAOBAB_RPC));

describe('Klay account encode account key API', () => {
    test('should return klay_encodeAccountKey', (done) => {

        let callbackOne = function (error, data, response) {
            expect(error).toBeNull();
            expect(data).toBeDefined();
            expect(/^0x[a-fA-F0-9]+/.test(data)).toBe(true);
            done();
        };
        const accountKey = {"keyType": 0, "key": {}}

        sdk.encodeAccountKey(accountKey, {}, callbackOne);
    });
});
